docs/4-advance.md

```diff
@@ -18,6 +19,13 @@ default: `"searchMethod": "fnmatch"`
 
 fnmatch is the behaviour (See criteria) by default and the fall back if this option is set incorrectly. `re` is the other choice if you want more flexibility to match criteria.
 
+## caseSensitive
+
+default: `"caseSensitive": "true"`
+
+If false, comparisons between strings/lists will be not case sensitive.
+It's only disabled when used with `"searchMethod": "fnmatch"`.
```

Comment:
I change the doc to:

> It can only be disabled when used with `"searchMethod": "fnmatch"`.

I tried your line with re and tests fail. I will stick to this now.

Reply:
Ohhh je vois. Quand vous dites "It", vous voulez dire "la sensibilité à la casse elle-même". Pour moi, cela ressemblait à "l'option de contrôler la sensibilité aux cas". Et pour le comprendre, j'ai essayé d'échanger les négations :

> It's only enabled when used with `"searchMethod": "re"

Ce qui, compte tenu de mon incompréhension de "ça", signifie

>  Case sensitivity is only enabled when used with `"searchMethod": "re"

Ce serait beaucoup plus clair si elle disait 

> This only has an effect if used with `"searchMethod": "fnmatch"`.

De plus, je suggères d'ajouter un autre exception si c'est utilisé avec `re`, jusqu'au moment ou les deux fonctionnent.

Reply:
@arnaudbore ^